include/enums.h

```diff
@@ -3783,11 +3783,11 @@ enum ActorEventFlags {
     ACTOR_EVENT_FLAG_SPIKY_FRONT            = 0x00010000, ///< Player takes spike damage from hammer attacks.
     ACTOR_EVENT_FLAG_20000                  = 0x00020000,
     ACTOR_EVENT_FLAG_ENCHANTED              = 0x00040000, ///< Actor glows and listens for the Star Beam event.
-    ACTOR_EVENT_FLAG_80000                  = 0x00080000,
+    ACTOR_EVENT_FLAG_GIANT_BOWSER_ENCHANTED = 0x00080000, ///< Actor has giant Bowser's Star Rod effect.
```

Comment:
I don't think this actually controls the effect, just the damage and event handling. I think ACTOR_EVENT_FLAG_STAR_ROD_ENCHANTED would be a better name.

Reply:
Name accepted, I will reword that description a little too.